site/sass/components/_stories-card.scss

```
@@ -0,0 +1,39 @@
+// Styles for stories card component.
+@use 'sass:math';
+
+.stories-card {
+  display: block;
+  text-decoration: none;
+
+  &__container {
+    @include elevation(0);
+    background-color: var(--white);
+    border-radius: 0.625rem;
+    display: flex;
+    flex-flow: column;
+    height: 100%;
+    overflow: hidden;
+    position: relative;
+    text-decoration: none;
+    width: 100%;
+    z-index: 1;
+
+    &:focus,
+    &:hover {
+      @include elevation(2);
+    }
+  }
+
+  &__image {
+    aspect-ratio: 16 / 10;
+    width: 100%;
```

```suggestion
    aspect-ratio: 16 / 10;
    object-fit: cover;
    width: 100%;
```

Looking at this with and without `object-fit`, I think it looks better with `object-fit`, otherwise stuff gets a little squished.